tools/DeviceExplorer/DeviceExplorer/MainForm.cs

```diff
@@ -1102,5 +1103,71 @@ private void devicesGridViewContextMenu_Opened(object sender, EventArgs e)
                 showDevicePropertiesToolStripMenuItem.Enabled = false;
             }
         }
+
+        #region Device Filtering
+        private void filterDevicesTextBox_TextChanged(object sender, EventArgs e)
+        {
+            filterDevices();
+        }
+
+        private void filterDevices() { filterDevices(filterDevicesTextBox.Text); }
```

Comment:
Nit: please start function names with caps. `FilterDevices()`. (See coding standards.)

I would not change the above function though since that is prefixed by the name of the textbox.

Reply:
Done!